packages/module/src/Message/Message.tsx

```diff
@@ -44,7 +44,7 @@ import ImageMessage from './ImageMessage/ImageMessage';
 import rehypeUnwrapImages from 'rehype-unwrap-images';
 import rehypeExternalLinks from 'rehype-external-links';
 import rehypeSanitize from 'rehype-sanitize';
-import { PluggableList } from 'react-markdown/lib';
+import { PluggableList } from 'unified';
```

Comment:
TS was suddenly complaining this is no longer exported for some reason? This is where it ultimately comes from in react-markdown.